### DISCUSSION TITLE
Navigation between founded entries inside text note for text search

### DISCUSSION THREAD

**Discussion by tty123** [standardnotes/forum#2520] [Suggestions] [answered]
When user has a large text note it will be really useful to navigate between founded text entries inside single note. Thanks

**Comment by atmoio** [comment]
Indeed this would be useful. We're still looking to implement an in-text search, so once we do that, this will likely be included.

**Comment by myreli** [comment, marked as answer]
Duplicate of https://github.com/standardnotes/forum/discussions/2498#discussioncomment-4963504
